modules/sno-du-conn-enabling-sctp.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-enabling-sctp_{context}"]
+= Enabling Stream Control Transmission Protocol (SCTP)
+
+SCTP is a key protocol used in RAN applications. This `MachineConfig` object adds the SCTP kernel module to the node to enable this protocol.
```

Comment:
While SCTP indeed is an important protocol, the doc should reflect that the decision to load the SCTP kernel module or not is down to CNF requirements

Reply:
Made this change.